.github/workflows/start-release.yml

```diff
@@ -174,10 +170,21 @@ jobs:
             git commit --message "Changelog for ${{inputs.version}}"
             echo "\`CHANGELOG.md\` has been updated on the release branch." >> "$GITHUB_STEP_SUMMARY"
           fi
+      - name: Tag master for next release
+        if: steps.check-existing.outputs.branch-exists == 'false'
+        run: |
+          git switch master
+          git commit --allow-empty --message \
+            "Bumping release tag to ${{needs.variables.outputs.next-release}}"
+          git tag --annotate --message "Upstream automation" \
+            "${{needs.variables.outputs.next-release}}" HEAD
```

Comment:
This must be
```suggestion
            "${{needs.variables.outputs.next-release}}.x" HEAD
```

Reply:
I've addressed this in https://github.com/stackrox/test-gh-actions/pull/158